Cargo.toml

```diff
@@ -22,25 +22,5 @@ slog-term = "2.4.1"
 unicode-normalization = "=0.1.5"
 crossbeam-channel = "0.4.2"
 
-[patch.crates-io]
```

Comment:
We need this for rebalancer, at least until we bump the version there. 

It has been a while, but IIRC believe the dependency on diesel is from sharkspotter's use of libmanta's MantaObjectShark.

Reply:
Ok, I think we could probably avoid having to specify it here, but similar to my other comment I'll just leave it as-is until there's a reason to update rebalancer with a new version.

Reply:
IIRC we couldn't compile rebalancer unless this was here because rebalancer also includes libmanta.  If you can remove this without breaking the rebalancer build than by all means do so.

Alternatively we could try updating everything to diesel 1.4.4 and see how that goes.